github-support/list-contributors.sh

```diff
@@ -20,16 +20,16 @@ CONTRIBUTORS_FILTERS="$DIR/FilteredNames.txt"
 
 cp $ALIASES $FILTERS $DIR && cd $DIR
 
-declare -a DEBEZIUM_REPOS=("debezium" "debezium-server" "debezium-operator" "debezium-connector-db2" "debezium-connector-cassandra" "debezium-connector-vitess" "debezium-connector-spanner" "debezium-ui" "container-images")
+declare -a DEBEZIUM_REPOS=("debezium" "debezium-server" "debezium-operator" "debezium-connector-db2" "debezium-connector-cassandra" "debezium-connector-vitess" "debezium-connector-spanner" "debezium-ui" "container-images" "debezium-connector-informix" "debezium-connector-ibmi" "debezium-connector-cockroachdb" "debezium-platform" "debezium-operator" "debezium-examples" "debezium-platform-stage" "debezium-platform-conductor" "debezium-connector-yugabytedb")
```

Comment:
I suggest removal of `"debezium-platform-stage" "debezium-platform-conductor" "debezium-connector-yugabytedb"` as they are not actively used.